modules/monitoring-creating-new-alerting-rules.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="creating-new-alerting-rules_{context}"]
+= Creating new alerting rules
+
+As a cluster administrator, you can create new alerting rules based on platform metrics. 
+These alerting rules fire alerts based on the values of chosen metrics.
```

Comment:
"Raise" or "trigger" alerts?

Reply:
We do use `fire` in other places, but I think `trigger` is better here. Changed.